tests/test_abstract_storage.py

```diff
@@ -0,0 +1,46 @@
+import asyncio
+from unittest import mock
+import json
+import time
+
+from aiohttp import web
+from aiohttp_session import (session_middleware,
+                             get_session, SimpleCookieStorage)
+
+
+def make_cookie(client, data):
+    session_data = {
+        'session': data,
+        'created': int(time.time())
+    }
+
+    value = json.dumps(session_data)
+    client.session.cookies['AIOHTTP_SESSION'] = value
+
+
+def create_app(loop, handler):
+    middleware = session_middleware(SimpleCookieStorage(max_age=10))
+    app = web.Application(middlewares=[middleware], loop=loop)
+    app.router.add_route('GET', '/', handler)
+    return app
+
+
+@asyncio.coroutine
+def test_max_age_also_returns_expires(test_client):
+
+    @asyncio.coroutine
+    def handler(request):
+        time.monotonic.return_value = 0.0
+        session = yield from get_session(request)
+        session['c'] = 3
+        return web.Response(body=b'OK')
+
+    with mock.patch('time.monotonic') as MockClass:
```

Comment:
Please use `m_monotonic` instead of `MockClass`.
Your patch is perfectly fine but all my projects use `m_*` naming for mocking.
I'd love to keep naming rules consistent.